--- src/Makefile.inc.gnu64.orig	2020-05-28 16:46:04.000000000 +0200
+++ src/Makefile.inc.gnu64	2023-05-14 13:19:54.730739954 +0200
@@ -4,7 +4,7 @@
 
 # defined compiler and options
 #
-INCL = -I. -I../common -I../common/dos2m
+INCL = -I. -I../common -I../common/dos2m -I$(LIBRARY_INC)
 
 %.o: %.F90
 	$(COMPILER) $(COMPFLAGS) -c $< -o $@
@@ -15,12 +15,11 @@
 
 # GNU Fortran 64 bits
 # -------------------
-F90 = gfortran
-F90FLAGS = -O2 -fbacktrace -mcmodel=medium -fdefault-double-8 -fdefault-integer-8 -fdefault-real-8
+F90FLAGS = -O2 -fbacktrace -mcmodel=medium -fdefault-double-8 -fdefault-integer-8 -fdefault-real-8 $(FFLAGS)
 
-LD = gfortran
+LD = $(F90)
 #LDFLAGS = -llapack -lblas -lpthread -static
-LDFLAGS = -L/usr/lib/atlas-base/atlas -llapack -lblas -Wl,--whole-archive -lpthread -Wl,--no-whole-archive
+LDFLAGS = -L$(LIBRARY_LIB) -llapack -lblas -lpthread -lgfortran -lm
 
 
 ARFLAGS = cru
